src/strategy/indicators/cmf.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { CMF } from './cmf'
+import { candles } from './spec.helper'
+
+fdescribe('CMF', () => {
+  fit('should calculate the correct cmf', () => {
+    const cmf = CMF.calculate(candles.slice(0, 19), 20)
+ //   expect(cmf).toEqual(1.099868227)
```

Comment:
Could you please fill the test case here? So we have an little test, which ensures that no one breaks this indicator.

Reply:
Also, remove the `f` from `fdescribe` and `fit`